spec/features/user_can_add_items_spec.rb

```diff
@@ -13,6 +13,22 @@
 # - Page must message users appropriately for invalid URLs / article numbers
 
 feature "User creates a new Ikea item" do
+  let(:user) do
+    User.create(
+      email: "asdf@asdf.com",
+      password: "asdf1234"
+  )
+  end
+
+  before(:each) do
+    user
+    visit root_path
+    click_link "Log in"
+    fill_in 'Email', with: "asdf@asdf.com"
+    fill_in 'Password', with: "asdf1234"
+    click_button "Log in"
+  end
```

Comment:
signing in a user for a test should be written out as a helper method, but for now `before(:each)` is fine